src/commands/server.cpp

```diff
@@ -22,6 +22,61 @@
 #include <vector>
 
 volatile std::sig_atomic_t eoserv_sig_abort = false;
+volatile std::sig_atomic_t eoserv_sig_reload = false;
+
+TimeEvent* shutdown_timer = nullptr;
+
+static void schedule_shutdown(const std::vector<std::string>& arguments, Command_Source* from, bool is_shutdown)
+{
+	if (shutdown_timer != nullptr)
+	{
+		from->ServerMsg("Shutdown/reload is already scheduled. Use $cancel to cancel pending shutdown/reload.");
```

Comment:
User message, i18n.

Reply:
Ugh I specifically didn't want to deal with it lol

How does BU handle translations for these? I don't see much of a point in adding it if we aren't also providing translations. I guess I could just drop it in google translate...

Reply:
You don't explicitly *need* to provide non-english translations for everything, it just means that the other internationalizations will be incomplete (which they are either way, if you're hardcoding english messages.)
I'd just put these strings in `lang.en.ini` and see if you can pester someone to translate them for you later. It's better than retroactively adding a bunch of `i18n.Format` calls everywhere and probably missing a bunch of them.

Reply:
Sidenote: if you want the other languages to fall back to the english language file instead of just the message key, you could always make some tweaks to the `I18N` class.

Reply:
Yeah, just saw that the other languages are very incomplete. I also forgot you did it for weddings too so it makes more sense for consistency to use i18n